src/panel/components/CodeHighlight.tsx

```diff
@@ -11,26 +11,29 @@ export const CodeHighlight: FC<
 > = ({ code, language, ...props }) => {
   const handleRef = useCallback(
     (ref: HTMLPreElement | null) => {
-      if (ref === null) {
+      if (!ref) {
         return;
       }
 
-      (ref.children[0] as HTMLElement).textContent = code;
```

Comment:
Issue was because this line was missing on the component below.

Making the change anyway to ensure we create a fresh code element on each update.